protocol-designer/src/components/steplist/ModuleStepItems.tsx

```diff
@@ -31,44 +32,64 @@ export const ModuleStepItemRow = (
   </PDListItem>
 )
 
-interface Props {
-  action?: string
+interface ModuleStepItemsProps {
   moduleType: ModuleType
   actionText: string
-  labwareNickname?: string | null
-  message?: string | null
+  moduleSlot?: string
+  action?: string
   children?: React.ReactNode
   hideHeader?: boolean
+  labwareNickname?: string | null
+  message?: string | null
 }
 
-export const ModuleStepItems = (props: Props): JSX.Element => {
-  const { t } = useTranslation('modules')
+export const ModuleStepItems = (props: ModuleStepItemsProps): JSX.Element => {
+  const {
+    moduleType,
+    actionText,
+    moduleSlot,
+    action,
+    hideHeader,
+    labwareNickname,
+    children,
+    message,
+  } = props
+  const { t } = useTranslation(['modules', 'application'])
   const [targetProps, tooltipProps] = useHoverTooltip({
     placement: 'bottom-start',
     strategy: TOOLTIP_FIXED,
   })
+  const moduleLongName = t(`module_long_names.${moduleType}`)
+
   return (
     <>
-      {!props.hideHeader && (
+      {!hideHeader && (
```

Comment:
hideHeader can be `undefined` so handling that case explicitly would be good. 

```suggestion
      {!Boolean(hideHeader) ?  () : null}
```